src/main/java/com/gmail/yuramitryahin/model/User.java

```diff
@@ -24,4 +27,6 @@
     @Column(unique = true, length = 20)
     private String email;
     private String password;
+    @OneToMany(fetch = FetchType.EAGER)
+    private Set<Role> role;
```

Comment:
Think if this relationship and FetchType are correct

Reply:
fixed